web/components/navbar.js

```diff
@@ -95,6 +95,11 @@ class Navbar extends React.Component {
                 <a>Workshop</a>
               </Link>
             </li>
+            <li className={styles.navitem}>
```

Comment:
On my screen, the objects tend to break lines which makes them end up under the navbar element now. Do you have a fix for this?

Reply:
Oh, another small ask - can you keep the link to the workshop as the rightmost item?

Reply:
Added workshop as a rightmost item and the navbar issue on reduced screen size was already present which we can take up in the later PRs.